frontend/src/app/aethel/aethel.component.ts

```diff
@@ -27,12 +28,17 @@ export class AethelComponent implements OnInit {
     faChevronRight = faChevronRight;
     faChevronDown = faChevronDown;
 
+    status$ = new Subject();
+
     constructor(
         private apiService: AethelApiService,
         private destroyRef: DestroyRef,
+        private statusService: StatusService
     ) {}
 
     ngOnInit(): void {
+        this.statusService.get().subscribe(status => this.status$.next(status.aethel));
```

Comment:
If I visit `/aethel` directly, there is no polling, and the input bar never becomes visible. Polling only starts if I visit `/spindle` first.

You'll probably want to copy some of the timer logic to this component as well, or better: move it to the `StatusService` and have both components share it.

Reply:
That's the intended behavior. When the app starts, the Aethel data is either loaded or not (if the file is missing). So there's no real reason to keep polling.